cloud/workspace-token/workspace_token.go

```diff
@@ -1,16 +1,20 @@
-package workspace
```

Comment:
I had to move this into a seperate file to prevent a cyclic import (caused by our auth package importing workspace and the new workspace token code importing organization)